.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest a new feature
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Describe the use case that is addressed by this feature.**
+The problem that this feature would solve.
+
+**Describe the solution you'd like**
```

Comment:
`**Describe the solution you'd like or the change(s) you'd like to propose**`

Reply:
@njmei what is the rationale?

Reply:
We have `bug report` and `feature request` but no `contribution proposal` (if external users want to contribute?) I was thinking that `feature request` could encapsulate both?

Reply:
I think it is a pretty reasonable suggestion though! I will add a field for volunteering :P